docs/CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## PLACEHOLDER FOR MERGED RELEASE DATE
+
+
+* Most projects can now be downloaded as a single object containing all libraries merged together, either in `SCE` or `AnnData` formats
```

Comment:
```suggestion
* When downloading data for an entire project, you will now have the option to download a single file with a single merged object containing all gene expression and metadata for all samples in that project. 
  * These merged objects are available as either `SingleCellExperiment` (`.rds` files) or `AnnData` (`.hdf5` files) objects.
  * This option is now available for most projects. 
  If the project you are interested in does not have this option, see our {ref}`FAQ on which projects can be downloaded as merged objects<faq:Which projects can I download as merged objects?>`. 
```
Just emphasizing that this is specific to downloading a project and that they will have to choose this option.